src/containers/MyNdla/useArenaModalActions.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Dispatch, SetStateAction, useCallback } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ArenaFormValues } from './Arena/components/ArenaForm';
+import { useCreateArenaTopic, useReplyToTopic } from './arenaMutations';
+import {
+  arenaTopicById,
+  useArenaCategory,
+  arenaCategoryQuery,
+} from './arenaQueries';
+
+const toArenaTopic = (topicId: number | undefined) =>
+  `/minndla/arena/topic/${topicId}`;
+
+interface ArenaModalProps {
+  setFocusId?: Dispatch<SetStateAction<number | undefined>>;
+  topicId?: number;
+}
+
+const useArenaModal = ({ setFocusId, topicId }: ArenaModalProps) => {
```

Comment:
Jeg synes abstraksjonsnivået her blir litt feil. Dette føles litt ut som starten på et såkalt "God object", som jeg helst vil unngå. Hvis vi først skal lage custom hooks synes jeg den kun skal omhandle et område, ikke fler. Kan heller ikke se for meg at dette er særlig gjenbrukbart dersom man skal ha forskjellige oppførsel på denne basert på hvor man bruker den.